src/common/pa_front.c

```diff
@@ -67,6 +67,7 @@
 #include <string.h>
 #include <stdlib.h> /* needed for strtol() */
 #include <assert.h> /* needed by PA_VALIDATE_ENDIANNESS */
+#include <stdbool.h>
```

Comment:
AFAIK portaudio is supposed to still support C89, while stdbool is C99. I'd replace the bool with an int to keep compatibility.

Reply:
Portaudio is already using stdbool  here:
https://github.com/PortAudio/portaudio/blob/68e963a990da19bb013133dcbad59c2ed8ea0cf9/src/hostapi/audioio/pa_unix_audioio.c#L54

Reply:
Host APIs are permitted to require newer versions as per #606.